experiments/long_runs/leaderboard/data_sources.jl

```diff
@@ -0,0 +1,142 @@
+import ClimaAnalysis
+
+# To add a variable for the leaderboard, add a key-value pair to the dictionary
+# `sim_var_dict` whose key is the short name of the variable and the value is a function
+# that returns a `OutputVar` Any preprocessing is done in the function which includes unit
+# conversion and shifting the dates.
+#
+# Then, add a key-value pair to the dictionary `obs_var_dict` whose key is the same short
+# name as before and the value is a function that takes in a start date and returns a
+# `OutputVar`. Any preprocessing is done in the function.
+#
+# Next, add a key-value pair to the dictionary `mask_dict` whose key is the same short name
+# as before and the value is a function that takes in a `OutputVar` representing simulation
+# data and a `OutputVar` representing observational data and returns a masking function or
+# `nothing` if no masking function is needed. The masking function is used to correctly
+# normalize the global bias and global RMSE.
+#
+# Finally, add a key-value pair to the dictionary `compare_vars_biases_plot_extrema` whose
+# key is the same short name as before and the value is a tuple of floats which determine
+# the range of the bias plots.
+
+# For plotting bias plots
+compare_vars_biases_plot_extrema = Dict(
+    "et" => (-0.00001, 0.00001),
+    "gpp" => (-8.0, 8.0),
+    "lwu" => (-40.0, 40.0),
+)
+
+# Dict for loading in simulation data
+sim_var_dict = Dict{String, Any}()
+
+for short_name in ["et", "lwu"]
+    sim_var_dict[short_name] =
+        () -> begin
+            sim_var = get(
+                ClimaAnalysis.SimDir(diagnostics_folder_path),
+                short_name = short_name,
+            )
+            sim_var =
+                ClimaAnalysis.shift_to_start_of_previous_month(sim_var)
+            return sim_var
+        end
+end
+
+for short_name in ["gpp"]
+    sim_var_dict[short_name] =
+        () -> begin
+            sim_var = get(
+                ClimaAnalysis.SimDir(diagnostics_folder_path),
+                short_name = short_name,
+            )
+            sim_var =
+                ClimaAnalysis.shift_to_start_of_previous_month(sim_var)
+            # converting from `mol CO2 m^-2 s^-1` in sim and `g C m-2 day-1` in obs
+            sim_var = ClimaAnalysis.convert_units(
+                sim_var,
+                "g m-2 day-1",
+                conversion_function = units -> units * 86400.0 * 12.011,
+            )
+            return sim_var
+        end
+end
+
+# Dict for loading in observational data
+obs_var_dict = Dict{String, Any}()
+obs_var_dict["et"] =
+    (start_date) -> begin
+        obs_var = ClimaAnalysis.OutputVar(
+            ClimaLand.Artifacts.ilamb_dataset_path(;
+                context = "evspsbl_MODIS_et_0.5x0.5.nc",
+            ),
+            "et",
```

Comment:
Do we rely on using the same short_name in the data and the diagnostic output from the simulation anywhere?

Reply:
or it seems like this line could be anything, as long as we set `obs_var_dict["et"]`

Reply:
We don't rely on the same short_name in the (observational?) data and the diagnostic output from the simulation.

It is like that because ClimaAnalysis sometimes can't correctly infers what variable is or if there is more than one variable in the .nc file.

The line can be anything as long as `obs_var_dict["et"]` is a function that takes in a start date and return a `OutputVar`.